src/glue/resources/table_columns.yaml

```diff
@@ -201,9 +201,9 @@ tables:
       - Name: startDate
         Type: string
       - Name: stepHistory
-        Type: struct<endDate:string,identifier:string,interactions:struct<controlEvent:string,parent:string,stepIdentifier:string,timestamp:string,userInteractionIdentifier:string,userInteractionNewStyleEncoder:boolean,value:string>,position:integer,practice:boolean,response:integer,responseTime:double,score:integer,startDate:string,type:string,wasInterrupted:boolean>
+        Type: struct<endDate:string,identifier:string,interactions:struct<controlEvent:string,parent:string,stepIdentifier:string,timestamp:string,userInteractionIdentifier:string,userInteractionNewStyleEncoder:boolean,value:string>,position:int,practice:boolean,response:int,responseTime:double,score:int,startDate:string,type:string,wasInterrupted:boolean>
```

Comment:
This should be `int`, not `integer`

Reply:
Good catch, is that because "integer" isn't supported?

Reply:
Glue tables want integer types to be written as `int`.